.storybook/webpack.config.js

```diff
@@ -6,6 +6,21 @@ const postcssCustomProperties = require('postcss-custom-properties');
 const postcssCustomMediaQueries = require('postcss-custom-media');
 const postcssPostcssColorModFunction = require('postcss-color-mod-function');
 
+const materialSources = [
+  'materials/borders.mod.css',
+  'materials/colors/base-colors.mod.css',
+  'materials/shadows/base-shadows.mod.css',
+  'materials/sizes.mod.css',
+  'materials/spacings.mod.css',
+  // tokens
+  'materials/tokens/backgrounds.mod.css',
+  'materials/tokens/borders.mod.css',
+  'materials/tokens/fonts.mod.css',
+  'materials/tokens/shadows.mod.css',
+  'materials/tokens/sizes.mod.css',
+  'materials/transitions.mod.css',
+];
```

Comment:
Well what @dferber90 and I talked about is getting rid of the scripts that generate these files and grabbing all of the variables from commented javascript files. I'm not sure how much it makes sense to invest in improving this array since my plan would be to remove it in the next or after-next iteration of #170